### PR TITLE
First scripts

### PR DESCRIPTION
**Pull request by kjrcda**
made first database erase scripts for mac and windows

### CODE REVIEW

File: webapp/Scripts/README.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ERASE
+To erase the database you need to run the Erase_DB script
+	Windows - Erase_DB.bat
+	Mac	- Earase_DB.sh
```
Comment (paymog) [inline webapp/Scripts/README.txt]: for mac we'd need to run `sh Erase_DB.sh`

**Comment by paymog** [comment]
can you rename `Scripts` to `scripts` (lowercase) and do the same with the files?

**Comment by kjrcda** [comment]
Made the file name changes but had to do it on GitHub site. Not sure how to do it with the folder name though...

**Comment by paymog** [comment]
http://www.patrick-wied.at/blog/rename-files-and-folders-with-git

**Pull request merged**